go.mod

```diff
@@ -1,11 +1,11 @@
 module github.com/aquasecurity/trivy-operator
 
-go 1.24.1
+go 1.24.2
 
 require (
 	github.com/CycloneDX/cyclonedx-go v0.9.2
 	github.com/aquasecurity/go-version v0.0.1
-	github.com/aquasecurity/trivy v0.61.0
+	github.com/aquasecurity/trivy v0.61.1-0.20250426053908-3ce7d59bb165
```

Comment:
Let's hold off for trivy v0.62.0 to be released as it's only a few days out now.

Reply:
sure! it's just for check, that we can merge the latest Trivy version

Reply:
actually yes - we can make a new PR when that happens. Let's get the tests fixed for this PR and merge it.

Reply:
ok, I'll do it

the error in tests was fixed https://github.com/aquasecurity/trivy/pull/8785

Reply:
done https://github.com/aquasecurity/trivy-operator/pull/2528/commits/9544fd4b71c1e029d171184f363615b28cb1f5ff